tests/test_tasks.py

```diff
@@ -43,6 +44,20 @@ def test_load_nox_module():
     assert noxfile_module.SIGIL == "123"
 
 
+def test_load_nox_module_expandvars():
+    # Assert that variables are expanded when looking up the path to the noxfile
+    # This is particular importand in Windows when one needs to use variables like
+    # %TEMP% to point to the noxfile.py
+    with mock.patch.dict(os.environ, {"RESOURCES_PATH": RESOURCES}):
+        if platform.system().lower().startswith("win"):
+            config = argparse.Namespace(noxfile="%RESOURCES_PATH%/noxfile.py")
+        else:
+            config = argparse.Namespace(noxfile="${RESOURCES_PATH}/noxfile.py")
```

Comment:
I'm surprised this branch doesn't trigger a coverage failure.

Reply:
Why should it? It's a test module.
Do you also track coverage of the test module's code?

Reply:
I'm unclear if `nox` tracks line coverage, but if it does it'd be easy to explicitly mark the uncovered windows line (or invoke it).

RE "why should it?" Dead code paths in test code are no fun either.

Reply:
So, very true....

Reply:
So, very true....